docs/usecases/extending_data_ecosystem.rst

```diff
@@ -0,0 +1,151 @@
+Git-annex uses a key-value pair to reference files. In the git-annex object tree,
+large files in datasets are stored as values while the key is generated from their
+contents and is checked into Git. The key is used to reference the location of the value
+in the object tree. The object tree (or keystore) with the data contents can
+be located anywhere – its location only needs to be encoded using a special remote.
+Therefore, thanks to the git-annex-globus-remote interface, Globus.org provides
+git-annex with locations information to retrieve values and access files content
+with the corresponding keys. To ultimately enable end users’ access to data,
+git annex registers Globus locations assigning them to Globus-specific URLs,
+such as `globus://datadet_id/path/to/file`, where each Globus URL is associated
+with a the key corresponding to the given file. The use of a Globus URL protocol
+is a fictitious mean to assign each file of the dataset a unique location and
+source and therefore, it is a wrapper for additional validation that is performed
+by the git-annex-globus-remote to check on the actual presence of the file within
+the Globus transfer file ecosystem. In other words, the ‘Globus URL’ is simply an
+alias of an existing file located on the web and specifically available in Globus.org.
+Registration of Globus URLs in git-annex is among the configuration procedures
+carried out on an administrative, system-wide level, and users will only deal
+with direct easy access of desired files.
+
+Therefore, Globus is configured to receive request for data access from the git-annex
+side and to respond back if data is available. Currently, git-annex-globus-remote
+only supports data download operations but it could potentially be useful for additional
+functionalities. When the globus-remote get initialized for the first time, the user
+has to authenticate to Globus.org using ORCHiD, Gmail or a specific Globus accounts:
+this step will enable git-annex to then initialize the globus-remote and establish the
+communication process. To use globus-remote, instructions are available here:
+https://github.com/CONP-PCNO/git-annex-remote-globus. To implement a special remote,
+guidelines specifying the standard communication protocol can be found here:
+https://git-annex.branchable.com/design/external_special_remote_protocol/
+
+
+Step-by-Step
+^^^^^^^^^^^^
+
+It always starts with a dataset:
+
+.. code-block:: bash
+
+   $ datalad install -r <dataset>
+   $ cd <dataset>
+
+Now we need to install the globus-special-remote, then authenticate to Globus.org if it is the first time
+by running the setup
+
+.. code-block:: bash
+
+   $ pip install git-annex-remote-globus
+   # if first time
+   $ git-annex-remore-globus setup
+
+We can see that most of the files in the dataset are annexed, highlighted in red. You can check
+the symlink for a given file by running
+
+.. code-block:: bash
+
+   $ ls ll path/to/file
```

Comment:
what is `ll` ?

Reply:
double-l (ll)? This command would show whether the file is annexed or not. It is used to compare before and after download

Reply:
it must be your alias or something like that -- it is not a standard option of `ls`.  I guess you meant `-l` - then indeed you could see if those are symlinks. **But** in cases of git-annex operating on crippled filesystems without symlinks support, or files being unlocked -- they wouldn't even be symlinks.